### DISCUSSION TITLE
[Feature Request - API] Include console ID on API_GetUserCompletedGames

### DISCUSSION THREAD

**Discussion by Bregann** [RetroAchievements/RAWeb#924] [Ideas]
Hey,

Could it be possible to include the Console ID on API_GetUserCompletedGames.php endpoint? Currently it only returns the console name but having the ID as well would be super useful

Thanks

**Comment by ScottFromDerby** [comment]
Added a PR here #925

**Reply by Bregann** [reply]
Awesome thank you!
